lib/classifier-reborn/bayes.rb

```diff
@@ -6,23 +6,26 @@
 
 module ClassifierReborn
   class Bayes
+    class CategoryNotFound < StandardError; end
```

Comment:
I'll add `Error`. Is there a compelling reason to choose one exception definition over another?